Sources/TuistLoader/Utils/ConfigLocator.swift

```diff
@@ -0,0 +1,30 @@
+import Foundation
+import TSCBasic
+import TuistCore
+import TuistSupport
+
+public protocol ConfigLocating {
+    ///  It traverses up the directory hierarchy until it finds a `Config.swift` file inside `tuist` dictionary
```

Comment:
`dictionary`? Did you mean `directory?`

Reply:
I mimicked documentation from `SetupLocator.swift` and `FileHandler.swift`

```swift
public protocol SetupLocating {
    /// It traverses up the directory hierarchy until it finds a `Setup.swift` file.
    /// - Parameter path: Path from where to do the lookup.
    func locate(at path: AbsolutePath) -> AbsolutePath?
}
```

```swift
/// It traverses up the directories hierarchy appending the given path and returning the
/// resulting path if it exists.
/// - Parameters:
///   - path: Relative path to append to each path in the hierarchy.
///   - from: Path to traverse the hierarchy from.
func locateDirectory(_ path: String, traversingFrom from: AbsolutePath) -> AbsolutePath?
```